new_platforms/docs/visualstudio_dev.md

```diff
@@ -42,48 +46,22 @@ flag when creating an enclave to automatically get the OP-TEE like behavior
+**Then to build the enclave for OP-TEE:**
 
-1. Right click the solution, select "Configuration Manager", create an ARM
-solution platform (if one doesn't already exist), and then create ARM
-configurations of your application project (and oehost if you include that
-directly).  Copy the configuration from the existing Win32 one.  Don't do
-this for your enclave project, as that needs to be built from a bash shell
-rather than in Visual Studio.
-2. Manually edit your application .vcxproj file to add the ability to
-compile for ARM, since Visual Studio cannot do it from the UI.  To do so, add the
-line "<WindowsSDKDesktopARMSupport\>true</WindowsSDKDesktopARMSupport\>"
-to each ARM configuration property group.  (See the sample apps'
-vcxproj file for examples.)
-3. For any new source files you add to your enclave project in
-Visual Studio, also add them to the sub.mk file in your optee subdirectory
-4. In your application project properties, under "Linker"->"Input", add
-rpcrt4.lib to the Additional Dependencies (All Configurations, ARM platform)
-which is required for string-to-UUID conversion, and remove any sgx libs.
-5. In your application project properties, update the Additional Include
-Directories to insert the $(NewPlatformsDir)include\optee\host and
-$(NewPlatformsDir)include\optee paths before the $(NewPlatformsDir)include path that you
-added earlier.
-6. On the destination machine, apply the uuids.reg file
+1. Using [Bash on Ubuntu on Windows](https://docs.microsoft.com/en-us/windows/wsl/about) shell, 
+cd to your enclave project's "optee" subdirectory.  If you added additional
+source files to your enclave project in Visual Studio, also add them to the 
+sub.mk in that directory.
+2. Do "make -f linux\_gcc.mak" to build the enclave.
+3. On the destination machine (if Windows), apply the uuids.reg file
 ("reg.exe import uuids.reg") and reboot.
 
 ## Debugging
 
-(Note: the information about the OPTEE-Simulation-Debug
-configuration is out of date and will be updated shortly.)
-
 **For SGX:** You can use Visual Studio for debugging, including the SGX
 simulator, that comes with the Intel SGX SDK.  Simply use the Debug
 configuration in Visual Studio if you have SGX-capable hardware, or
 the SGX-Simulation-Debug configuration in Visual Studio for software emulation.
 
 **For TrustZone:** You can use a basic software emulation environment with OP-TEE
-by creating and using a OPTEE-Simulation-Debug configuration in Visual Studio,
-as follows...
-
-1. Create a new configuration (say, for x86 and called OPTEE-Simulation-Debug)
-based on the Debug configuration. A new configuration can be created
-inside Visual Studio by right clicking on the solution, and accessing
-the "Configuration Manager" screen.
-2. In the "Configuration Properties->"Debugging", make the working directory
-be the directory your enclave is built in (usually "$(OutDir)").
+by creating and using the OPTEE-Simulation-Debug configuration in Visual Studio.
```

Comment:
`ImportEnclaveCommand.cs:Execute` appears to create the `OPTEE-Simulation-Debug` configuration.

Reply:
Updated